selenium exercises/ex1.py

```diff
@@ -0,0 +1,22 @@
+from selenium.webdriver import Chrome
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+from time import sleep
+from selenium.webdriver.chrome.service import Service as chromeService
```

Comment:
uses classes for all exercises, avoid leaving extra lines between code, only one line should be left between each new function